addons/medical/stringtable.xml

```diff
@@ -18,6 +18,12 @@
             <Turkish>ACE Medikal</Turkish>
             <Japanese>ACE 医療</Japanese>
         </Key>
+        <Key ID="STR_ACE_Medical_EnableDropWeaponOnUnconscious_DisplayName">
+            <English>Drop weapon on unconscious</English>
```

Comment:
Should be Title Case and mention that this is **chance**

Reply:
Drop Weapon When Falling Unconscious

Drop Weapon When Losing Consciousness

🤔

Reply:
Drop Weapon On Loss of Consciousness